ArchiSteamFarm/Commands.cs

```diff
@@ -1156,7 +1192,12 @@ internal sealed class Commands {
 				return FormatBotResponse(Strings.BotNotConnected);
 			}
 
-			(bool success, string output) = await Bot.Actions.Loot(wantedTypes: Bot.BotConfig.LootableTypes).ConfigureAwait(false);
+			ulong targetSteamID = Bot.Actions.GetFirstSteamMasterID();
```

Comment:
So why exactly did you decide that extracting code from a single function and duplicating it in 4 different places was a good idea?

Previous approach was fine and much better than your existing one. I still have no clue why you won't write a generic action responsible for sending a trade and call it with different arguments from different loot/transfer commands. Right now this code is worse than before a PR.